platform/contracts/admin/src/msg.rs

```diff
@@ -21,9 +19,8 @@ pub struct InstantiateMsg {
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
 #[serde(rename_all = "snake_case", deny_unknown_fields)]
 pub struct MigrateMsg {
-    pub protocol_name: String,
-    pub network_name: String,
-    pub dex_admin: Addr,
+    pub migration_spec: String,
+    pub post_migration_execute: String,
```

Comment:
what migration messages are these for? The Admin itself or the contracts? You use them as the latter .... in the `migrate` endpoint that should be about the Admin only. 
In addition, if they were for the Admin then we would have them in a structured form, not in a regular json String.